ocpmodels/datasets/lmdb_dataset.py

```diff
@@ -0,0 +1,164 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+import bisect
+import logging
+import math
+import pickle
+import random
+import warnings
+from pathlib import Path
+
+import lmdb
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+from torch_geometric.data import Batch
+
+from ocpmodels.common import distutils
+from ocpmodels.common.registry import registry
+from ocpmodels.common.utils import pyg2_data_transform
+
+
+@registry.register_dataset("oc20_lmdb")
```

Comment:
Should we rename this registration to `lmdb` as well? Or would that conflict with the `lmdb` package?